TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/AssetEditor.js

```diff
@@ -134,7 +134,7 @@ function(Ember, $, FileUpload, template, SecondaryInspectorController, Utility,
 				assetIdentifiers = [$.parseJSON(value)];
 			}
 
-			if (assetIdentifiers.length > 0) {
+			if (assetIdentifiers.length > 0 && assetIdentifiers[0] != null) {
```

Comment:
this works, but it's an odd place to check for it.. also should be `!== null` if we keep it here